myhoard/basebackup_operation.py

```diff
@@ -118,6 +118,7 @@ def create_backup(self):
 
         self.data_directory_size_end, self.data_directory_filtered_size = self._get_data_directory_size()
         self._update_progress()
+        self.stats.gauge_float("myhoard.basebackup.estimated_progress", 100)
```

Comment:
nit: I would probably inject the `100` into `_update_progress` (and not calculate it if it was provided) to avoid duplicating the stat name